### PR TITLE
Rename icw_extensions_{gpcloud,gps3ext} + add oracle7 test

### PR DESCRIPTION
**Pull request by amilkh**
Developer pipeline:
https://dev.ci.gpdb.pivotal.io/teams/main/pipelines/icw-gps3ext-oracle7

Co-authored-by: Adam Lee <ali@pivotal.io>
Co-authored-by: Amil Khanzada <akhanzada@pivotal.io>

### CODE REVIEW

File: concourse/scripts/gpcheckcloud_tests_gpcloud.bash [hunk 1]
```diff
@@ -30,7 +30,8 @@ function setup_gpadmin_user() {
 
 function _main() {
 	time configure
-        sed -i s/1024/unlimited/ /etc/security/limits.d/90-nproc.conf
+        sed -i s/1024/unlimited/ /etc/security/limits.d/*-nproc.conf # centos6
+        sed -i s/4096/unlimited/ /etc/security/limits.d/*-nproc.conf # centos7
```
Comment (amilkh) [inline concourse/scripts/gpcheckcloud_tests_gpcloud.bash]: Does anybody know a better idea ?

**Comment by amilkh** [comment]
@lavjain Adam mentioned that the `icw_extensions_{gpcloud,gps3ext}` is now owned by the UD team, but was helpful to pair with me and make this initial change.

We need to change references of gpcloud to gps3ext in other places as well.

Also, we should probably add a `gpcheckcloud_tests_gpcloud_oracle7`

**Comment by adam8157** [comment]
The tests for Oracle 7 on 5X_STABLE is from a customer's request. Does master branch need this too?

Not only this time Toolsmiths wants to change the task name, I remember there was a discussion about renaming gpcloud back to gps3ext or another name, because it conflicts with other projects, I would like to leave that call to the owner team to make.

File: concourse/scripts/gpcheckcloud_tests_gpcloud.bash (hunk 1 above)
Comment (lisaoakley) [reply concourse/scripts/gpcheckcloud_tests_gpcloud.bash]: The way we did it for the mapr and gphdfs regression tests is to include a conditional on the OS version (https://github.com/greenplum-db/gpdb/pull/5374/commits/676f09d21521a861cd45ff253232c2b591b870f5)

The OS version should already be provided to this script from the pipeline level

**Comment by Eulerizeit** [comment]
Is there a PR in Master for this?  We need to do it there first.

Rob

**Comment by amilkh** [comment]
Alex, Rob, this it not needed for master branch yet. Customer will use gpdb 5 (not gpdb 6 master).

**Comment by Eulerizeit** [comment]
By doing stuff in 5.x and not in master is how we drop things and forget to
put them in places. The only way we can keep our repos organized and neat
is by following the process of merging into master and then backporting to
which ever effected branch. We have been bitten by this several times
already.

To say that the customer will use 5 and not 6 is tautological since 6
hasn't been released. OTOH we need to make sure that all changes are
landing in master so that when it is released we aren't missing, support,
features, bug fixes CVEs etc.

-- Rob

**Comment by lavjain** [comment]
@amilkh The UD team is not aware of the transition of s3ext component. Let us discuss this with @edespino and @kongyew

**Comment by danielgustafsson** [comment]
> On 2 Aug 2018, at 18:55, Robert Eckhardt <notifications@github.com> wrote:
> 
> By doing stuff in 5.x and not in master is how we drop things and forget to
> put them in places. The only way we can keep our repos organized and neat
> is by following the process of merging into master and then backporting to
> which ever effected branch. We have been bitten by this several times
> already.

+1. I will use the veto I don’t have to -1 anything going to 5X and not master. The only exception is a bugfix for a bug only existing on 5X.

**Comment by schubert** [comment]
@adam8157 @amilkh - @Eulerizeit and @danielgustafsson are right. There is no reason this shouldn't go in master first. I know this is important but no reason to rush this and things only go to a *_STABLE branch first if its a fix that would only apply to that branch because master diverges in implementation. That is not the case here.

Please re-PR on master with the suggestions from here and then pull back into 5X_STABLE.

Thanks. cc @yydzero @macroyuyang @edespino @pivotal-mike

**Comment by macroyuyang** [comment]
Agreed commit should go to master first except specific bug fix

**Comment by amilkh** [comment]
Thanks!

@adam8157 we will take care of re-basing on master before merging this to 5X_STABLE, but thanks for pairing last night!

**Comment by adam8157** [comment]
Close, will re-do this on master after oracle7 resources merged.

Feel free to remind me if I forget.

**Pull request closed without merging**